File: MathGameClient/src/app/models/gameExpression.ts
import { AnswerStatus } from "../enums/AnswerStatus";

export interface GameExpression {
    id: number;
    mathExpression: string;
    result: number;
    gameSessionId: number;
    gameSession: any;
    answerStatus: AnswerStatus;
    answeredFromPlayer: string;
    answer: number;
  }
  

File: MathGameServer/MathGame_Domain/EntityModels/GameExpression.cs
using MathGame_Domain.Enums;
using Newtonsoft.Json;
using System.Text.Json.Serialization;

namespace MathGame_Domain.EntityModels
{
    public class GameExpression : BaseEntity
    {
        [JsonPropertyName("mathExpression")]
        public string? MathExpression { get; set; }

        [JsonPropertyName("result")]
        public double Result { get; set; }

        [JsonPropertyName("gameSessionId")]
        public int GameSessionId { get; set; }

        [JsonPropertyName("gameSession")]
        public GameSession GameSession { get; set; }

        [JsonPropertyName("answerStatus")]
        public AnswerStatus AnswerStatus { get; set; }

        [JsonPropertyName("answeredFromPlayer")]
        public string? AnsweredFromPlayer { get; set; }

        [JsonPropertyName("answer")]
        public double Answer { get; set; }
    }
}
